README.md

```diff
@@ -1,46 +1,64 @@
-Opolo_Global_Page
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Opolo_Global_Page
 For this repository, I've created html templates and styling files in order to built a responsive conference website, which can help to easy explorate profiles from differents devices.
 
-Built With 🔨
-HTML
-CSS
-JS
-Git & Github
-Live Demo
-Live Demo Link
+## Built With 🔨
+
+- HTML
+- CSS
+- JS
+- Git & Github
+
+## Live Demo (if available)
+Live Demo [Link](https://engruugbejimmy.github.io/Opolo_Conference_Webpage/)
+
+## Video Demo
+Video Introduction[Link](https://www.loom.com/share/9d7c65a1c21e4867b5cf2a77c12c010d)
+
+## Install
 
-Install
 To get a local copy up and running follow these simple example steps.
 
-Open terminal
-Clone this project by the command:
-$ git clone git@github.com:EngruugbeJimmy/Opolo_Conference_Webpage.git
+- Open terminal
+- Clone this project by the command:
+- $ git clone git@github.com:EngruugbeJimmy/Opolo_Conference_Webpage.git
 
 Then go to the main folder using the next command:
-$ cd Portfolio-setup-and-mobile-version-skeleton.git
+- $ cd Portfolio-setup-and-mobile-version-skeleton.git
 
 Finally, run diplay the index.html file in your local browser, or also, you can use Live Server in Visual Studio Code.
 
-Prerequisites
-IDE to edit and run the code (We use Visual Studio Code 🔥).
-Git to versionning your work.
-Usage
-For anyone who wants to practice html5 or/and css3
-For anyone who wants to create his own portfolio.
-Authors
+## Prerequisites
+- IDE to edit and run the code (We use Visual Studio Code 🔥).
+- Git to versionning your work.
+
+## Usage
+- For anyone who wants to practice html5 or/and css3
+- For anyone who wants to create his own portfolio.
+
+
+## Authors
 👤 ABAH JAMES
 
-GitHub: @EngruugbeJimmy
-LinkedIn: @abah-james-ugbede
+- GitHub: [@EngruugbeJimmy](https://github.com/EngruugbeJimmy)
+- LinkedIn: [@abah-james-ugbede](https://www.linkedin.com/in/abah-james-ugbede-356982159/)
+
 🤝 Contributing
-Contributions, issues, and feature requests are welcome!
+- Contributions, issues, and feature requests are welcome!
 
 Show your support
-Give a ⭐️ if you like this project!
-
-Acknowledgments
-Hat tip to anyone whose code was used 🔰
-Inspiration 💘
-Microverse program ⚡
-My standup team 🏹
-My family's support 🙌
\ No newline at end of file
+- Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Hat tip to anyone whose code was used 🔰
+- Inspiration 💘
+- Microverse program ⚡
+- My standup team 🏹
+- My family's support 🙌
```

Comment:
Kindly make sure that your README file gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly add this to your README file. :pray: :pray: